.github/ISSUE_TEMPLATE/new_feature

```diff
@@ -0,0 +1,13 @@
+## Desired Behavior
+
+Describe desired behavior for improvements/enhancements.
+What is the problem we want to solve?
+What is the plan?
+
+## Actual Behavior
```

Comment:
`Actual` is surely correct, but maybe, since we're not addressing bugs, we should use `Current` instead?

Reply:
Makes sense, done.